products/sle/main.pm

```diff
@@ -784,7 +784,12 @@ sub load_consoletests {
     }
     #have SCC repo for SLE product
     if (have_scc_repos()) {
-        loadtest "console/yast_scc";
+        if (get_var("USE_PROXYSCC")) {
```

Comment:
IMHO quite misleading because all tests which have SCC_REGISTER set and have a custom regurl set *use proxyscc*. How about just reuse the variable SCC_REGISTER that is already used within `have_scc_repos()"? For example use "suseconnect" as a value to start registration using SUSEConnect, if "yast" use the yast module.